tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,274 @@
+
+log = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+# Project name
+PROJECT_NAME = "ocs-372"
+
+PRJ = ocp.OCP(kind='Project')
+
+# yaml path
+TEMPLATES_DIR = "/tmp/"
+
+template_data = {}
+
+# Service
+OCP = ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# CephBlockPool
+CBP = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Storage class
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Secret
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+
+# PVC
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=PROJECT_NAME
+)
+
+# Pod
+POD = ocp.OCP(
+    kind='Pod', namespace=PROJECT_NAME
+)
+
+
+def create_cephblock_pool(pool_name):
+
+    """
+    Create a cephblockpool
+    """
+
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'CephBlockPool.yaml', TEMPLATES_DIR, _templating,
+        template_data, template_dir='CSI/rbd'
+    )
+
+    # Validate cephblock created on oc
+    assert CBP.get(f'{pool_name}')
+
+    # Validate cephblock created on ceph
+    _rc = False
+    pools = ocp.exec_ceph_cmd(ceph_cmd="ceph osd lspools")
+    for pool in pools:
+        if pool['poolname'] == pool_name:
+            _rc = True
+    assert _rc, f"Pool: {pool_name} wasn't created'"
+
+
+def create_storageclass(sc_name, pool_name):
+    """
+    Create a storage class
+    """
+    template_data['rbd_storageclass_name'] = sc_name
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'storageclass.yaml', TEMPLATES_DIR, _templating,
+        template_data, "CSI/rbd"
+    )
+
+    # Validate storage class created
+    assert SC.get(f'{sc_name}')
+
+
+def create_secret(secret_name, pool_name):
+    """
+    Create secret to store username and password
+    """
+
+    # Get the keyring for admin
+    admin = ocp.getbase64_ceph_secret("client.admin")
+    template_data['base64_encoded_admin_password'] = admin
+
+    # Create a user=kubernetes
+    assert ocp.exec_ceph_cmd(
+        ceph_cmd=f'ceph auth get-or-create-key client.kubernetes mon '
+        f' "allow profile rbd" osd "profile rbd pool={pool_name}"')
+
+    # Get the keyring for user=kubernetes
+    kubernetes = ocp.getbase64_ceph_secret("client.kubernetes")
+    template_data['base64_encoded_user_password'] = kubernetes
+
+    template_data['csi_rbd_secret'] = secret_name
+
+    create_oc_resource(
+        'secret.yaml', TEMPLATES_DIR, _templating, template_data, "CSI/rbd"
+    )
+
+    # Validate secret is created
+    assert SECRET.get(f'{secret_name}')
+
+
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    cmd = "ceph status"
+    pods = ocp.exec_ceph_cmd(cmd)
+    assert pods is not None
+    used = pods['pgmap']['bytes_used']
+    GB = (1024 ** 3)
+    used_in_gb = used / GB
+    return used_in_gb
+
+
+def create_project(project_name):
```

Comment:
you have new_project() under ocp.py
If you already have that project in your cluster it means the teardown needs more fixes